pipeline/jobs/test-performance.yaml

```diff
@@ -0,0 +1,63 @@
+parameters:
+  - name: displayName
+    type: string
+
+  - name: dependsOn
+    type: object
+    default: ''
+
+  - name: environment
+    type: string
+
+  - name: subscription
+    type: string
+
+  - name: variables
+    type: object
+
+jobs:
+- job: PerformanceTest
+  dependsOn:
+  - ${{ if parameters.dependsOn }}:
+    - ${{ parameters.dependsOn }}
+  variables:
+  - group: HMI-PERF-BUILD-${{ parameters.environment }}
+  - name: storageContainer
+    value: ${{ parameters.variables.storageContainer }}
+  displayName: ${{ parameters.displayName }}
+  steps:
+
+  - checkout: fh-hmi-performance
+  - task: replacetokens@3
+    inputs:
+      rootDirectory: '$(Build.sourcesdirectory)/src/test/resources/properties'
+      targetFiles: "${{ parameters.environment }}.application.properties"
+      encoding: 'auto'
+      writeBOM: true
+      actionOnMissing: 'warn'
+      keepToken: false
+      tokenPrefix: '${'
+      tokenSuffix: '}'
+      useLegacyPattern: false
+      enableTransforms: false
+      enableTelemetry: true
+  - task: Maven@4
+    displayName: 'Maven pom.xml'
+    inputs:
+      mavenPomFile: 'pom.xml'
+      mavenOptions: '-Xmx3072m'
+      javaHomeOption: 'JDKVersion'
+      jdkArchitectureOption: 'x64'
+      publishJUnitResults: false
+      goals: 'clean compile gatling:test -Dgatling.simulationClass=simulations.SIT.CreateHMCSimulation -Denv=${{ parameters.environment }}'
+  - template: ../steps/zip-test-results.yaml
+    parameters:
+      zipName: test_performance
+      directory: /target/gatling
+  - template: ../steps/publish-test-results.yaml
```

Comment:
I see we are pushing the results to a storage account. Do we need to be mindful of how many times this will run and how much storage it will take?
Can we not just publish them to the pipeline?

Reply:
this is just what was already in place. I have no idea why we are doing it this way - it was also not really working correctly. I will check with the testers to see what they want.

Reply:
I am going to resolve this, but raise a ticket to look in to this as a different change.

Reply:
https://tools.hmcts.net/jira/browse/HMIS-1134